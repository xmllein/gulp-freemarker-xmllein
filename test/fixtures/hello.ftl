Hello
${name}

<#include "msg.ftl" />