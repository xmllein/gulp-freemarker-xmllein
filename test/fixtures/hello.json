{
	"file": "hello.ftl",
	"data": {
		"name": "World"
	}
}
