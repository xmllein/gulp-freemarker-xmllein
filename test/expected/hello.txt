Hello
World


particle page of World.
